### PR TITLE
add bcl2fastq version in x_flowcell DB

### PR DESCRIPTION
**Pull request by senthil10**
This information is required to put in the project report.

### CODE REVIEW

File: taca/illumina/Runs.py [hunk 1]
```diff
@@ -38,34 +38,41 @@ def __init__(self, run_dir, configuration):
...
+
+
     def _get_sequencer_type(self):
         if self.sequencer_type:
             return self.sequencer_type
         else:
             raise RuntimeError("sequencer_type not yet available!!")
+    
+    
+    def _get_run_parser_obj(self, configuration):
```
Comment (Galithil) [inline taca/illumina/Runs.py]: you can't have a get method that does not return anything. that's a set method.

**Comment by Galithil** [comment]
other than the get/set mixup, looks fine.

File: taca/illumina/Runs.py (hunk 1 above)
Comment (senthil10) [inline taca/illumina/Runs.py]: ah ha!! dint know that :sweat_smile: will change it :+1:

**Comment by senthil10** [comment]
@Galithil : Changed :+1:

**Comment by vezzi** [comment]
Looks fine to me

**Pull request merged**